packages/rollup.config.js

```diff
@@ -0,0 +1,7 @@
+"use strict";
+
+const sourcemaps = require("rollup-plugin-sourcemaps");
```

Comment:
What is the idea of having this file? Who will use this and how? Because I see that we already have a rollup config in packages/stark-core which is more complete